client/src/components/Register/tests/Introduction-test.js

```diff
@@ -0,0 +1,84 @@
+/* global jest, describe, it, expect */
+
+// eslint-disable-next-line no-unused-vars
+import fetch from 'isomorphic-fetch';
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Introduction from '../Introduction';
+import { loadComponent } from 'lib/Injector'; // eslint-disable-line
+
+Enzyme.configure({ adapter: new Adapter() });
+
+window.ss = {
+  i18n: { _t: (key, string) => string },
+};
+
+const fetchMock = jest.spyOn(global, 'fetch');
+const handleContinueMock = jest.fn(() => true);
+const handleSkipMock = jest.fn(() => true);
+
+describe('Introduction', () => {
+  beforeEach(() => {
+    fetchMock.mockImplementation(() => Promise.resolve({
+      status: 200,
+      json: () => Promise.resolve({}),
+    }));
+
+    fetchMock.mockClear();
+    handleContinueMock.mockClear();
+    handleSkipMock.mockClear();
+  });
+
+  it('does not render a skip button by default', () => {
```

Comment:
Ideally you wrap these descriptions inside describe() callbacks for the method they're describing, e.g. `describe('render()', () => {`

Reply:
I've adjusted all these tests to relate to either `render()` or `ActionList`.